pkg/koordlet/resmanager/cpu_burst.go

```diff
@@ -367,6 +368,7 @@ func (b *CPUBurst) applyCFSQuotaBurst(burstCfg *slov1alpha1.CPUBurstConfig, podM
 				pod.Namespace, pod.Name, containerStat.Name, finalOperation, deltaContainerCFS, err)
 			continue
 		}
+		metrics.RecordLSPodCFSBurstQuota(pod.Namespace, pod.Name, containerStat.ContainerID, containerStat.Name, float64(containerTargetCFS))
```

Comment:
Metrics belonging to pods should be reset or GCed at each turn. Otherwise, the old metrics for pods no longer burst would keep in memory until the koordlet restarts.